### PR TITLE
6.9.2/rel-notes

### PR DESCRIPTION
**Pull request by jhenderson-pro**
## Description

Creating PR for updating release notes on the 6.9.2 release.

### CODE REVIEW

File: content/sensu-go/6.9/release-notes.md [hunk 1]
```diff
@@ -113,6 +114,17 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+## 6.9.2 release notes
+
+**March 8, 2023** &mdash; The latest release of Sensu Go, version 6.9.2, is now available for download.
+
+Sensu Go 6.9.2 fixes a UI crash on the Service Component detail page. Additionally we added a GraphQL validator for query node depth.
+
+**FIXES:**
+
+- ([Commercial feature][303]) In the web UI, added GraphQL validator for query node depth.
+- ([Commercial feature][303]) In the web UI, fixed crash on BSM Service Component detail page.
```
Comment (asachs01) [inline content/sensu-go/6.9/release-notes.md]:
```suggestion
- ([Commercial feature][303]) In the web UI, fixed an issue causing crashes when viewing BSM Service Component detail pages.
```

File: content/sensu-go/6.9/release-notes.md [hunk 2]
```diff
@@ -113,6 +114,17 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
 
+## 6.9.2 release notes
+
+**March 8, 2023** &mdash; The latest release of Sensu Go, version 6.9.2, is now available for download.
+
+Sensu Go 6.9.2 fixes a UI crash on the Service Component detail page. Additionally we added a GraphQL validator for query node depth.
+
+**FIXES:**
+
+- ([Commercial feature][303]) In the web UI, added GraphQL validator for query node depth.
```
Comment (asachs01) [inline content/sensu-go/6.9/release-notes.md]: This is the XSS issue, isn't it? We should probably be a bit more descriptive about what we're actually fixing.

**Review by asachs01 — CHANGES_REQUESTED** [review]
Needs some massaging on the Graphql comment. That should make its way into the Summary.

**Review by jhenderson-pro — COMMENTED** [review]
I've added additional context in the release notes

**Review by asachs01 — APPROVED** [review]
👍

**Pull request merged**